cmake/scripts/CreateGRF.cmake

```diff
@@ -39,6 +39,20 @@ foreach(NFO_LINE IN LISTS NFO_LINES)
     endif()
 endforeach()
 
-execute_process(COMMAND ${NFORENUM_EXECUTABLE} -s sprites/${GRF_SOURCE_FOLDER_NAME}.nfo)
-execute_process(COMMAND ${GRFCODEC_EXECUTABLE} -n -s -e -p1 ${GRF_SOURCE_FOLDER_NAME}.grf)
+execute_process(COMMAND ${NFORENUM_EXECUTABLE} -s sprites/${GRF_SOURCE_FOLDER_NAME}.nfo RESULT_VARIABLE RESULT)
+if(RESULT)
+    if(NOT RESULT MATCHES "^[0-9]*$")
```

Comment:
what does this regex capture? Maybe a comment of some kind might help future-us, but mainly I am just curious :D

Reply:
`RESULT` can be "an integer return code from the last child or a string describing an error condition" as stated in CMake documentation. So if it's not an integer it didn't run at all and there's probably an issue with the executable itself.

Reply:
Smart :D